src/components/MapBox.tsx

```diff
@@ -1,21 +1,33 @@
-import type { Map, LatLngTuple } from 'leaflet';
+import { Map, LatLngTuple, Icon } from 'leaflet';
 import { Box } from 'grommet';
 import { useCallback, useEffect, useMemo, useState } from 'react';
 import { MapContainer, Marker, Popup, TileLayer, ZoomControl } from 'react-leaflet';
 import Logger from '../utils/logger';
-import L from 'leaflet';
-import { useAppState } from '../store';
+import { useAppDispatch, useAppState } from '../store';
+import { AccommodationLocation } from '@windingtree/glider-types/types/derbysoft';
+import icon from 'leaflet/dist/images/marker-icon.png';
+import { FacilityRecord } from '../store/types';
 
 const logger = Logger('MapBox');
 const defaultZoom = 13;
 
-const pinIcon = new L.Icon({
-  iconUrl: require('../images/pin.svg'),
-  iconRetinaUrl: require('../images/pin.svg'),
-  iconSize: new L.Point(20, 20),
-  className: 'leaflet-div-icon'
+// const pinIcon = new L.Icon({
+//   iconUrl: require('../images/pin.svg'),
+//   iconRetinaUrl: require('../images/pin.svg'),
+//   iconSize: new L.Point(20, 20),
+//   className: 'leaflet-div-icon'
+// });
+
+const pinIcon = new Icon({
+  iconUrl: icon,
+  iconSize: [25, 40]
 });
 
+// TO-BE-REMOVED: interface for workaround on coordinates
```

Comment:
I added this work-around interface to add support for the current `win.so` backend interface